### PR TITLE
Big clean up of decommed rules

### PR DESCRIPTION
**Pull request by dividehex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,121 +1,31 @@
...
+Actions are deployed with Terraform
 
+To deploy to the Auth0 dev tenant
+Make sure `AUTH0_DOMAIN`, `AUTH0_CLIENT_ID`, and `AUTH0_CLIENT_SECRET` are set for the dev tenant.
 ```
-$ python3 -m venv venv
-$ source venv/bin/activate
-$ pip install -r requirements.txt
-$ uploader_rules.py <args>
+$ cd tf
```
Comment (bheesham) [inline README.md:18]:
```suggestion
$ cd terraform
```

As a side-note: if we want to encourage copy/pasting, I'd suggest removing the `$ ` prefix.

**Review by bheesham — APPROVED** [review]
Solid. 🎉

File: README.md (hunk 1 above)
Comment (dividehex) [reply README.md:18]: This PR renames the `terraform` dir to `tf` so the change here is accurate.

Comment (dividehex) [reply README.md:18]: I agree with removing the `$` prefix but we can do that in future PRs

Comment (bheesham) [reply README.md:18]: ah, whoops, missed that rename.

**Pull request merged**